pages/api/graphql-rest.page.ts

```diff
@@ -123,6 +127,15 @@ class MpdxRestApi extends RESTDataSource {
     return getTaskAnalytics(data);
   }
 
+  async getCoachingAnswerSets(accountListId: string, completed?: boolean) {
+    debugger;
+    const { data }: { data: CoachingAnswerSetResponse[] } = await this.get(
+      `coaching/answer_sets?filter[account_list_id]=${accountListId}&filter[completed]=${completed}&include=answers,answers.response&sort=-completed_at`,
```

Comment:
@OzzieOrca @TheNoodleMoose This is the problem spot.  I need more help debugging this.  Christian and I figured out that the `answers` and `questions` relationships only contain an id, no other associated data.  It should just be as simple as explicitly including those fields in this query, but nothing I've tried has worked.  I tried adding `fields[answers]=response`, `fields[coaching_answers]=response`, and `include=answers.answers.response`, still no data.  Any thoughts?